test/run_test.go

```diff
@@ -137,7 +137,7 @@ func TestCgoOk(t *testing.T) {
 			"--timeout=3m",
 			"--enable-all",
 			"-D",
-			"nosnakecase,gci",
+			"nosnakecase,gci,gofactory",
```

Comment:
need to rollback

Reply:
I added because of linter error `Use factory for unsafe.Pointer`.

Should I add option to exclude checking of std libraries in gofactory?

Reply:
> Should I add option to exclude checking of std libraries in gofactory?

Nice idea, but I think in case of `exclude-std=false` we will receive false positive anyway, because a lot of std types could be (or must be) used without factory by design. Like `unsafe.Pointer` or `sync.Mutex`.

**Zero value is useful** 🙂 

Look at
```
$ cd $GOROOT/src
$ gofactory -json ./... | jq '.[] | .[] | .[] .message' | sort | uniq
```